examples/configs/instantiate.yaml

```diff
@@ -0,0 +1,13 @@
+# python examples/real_convolve.py -cn exp1
```

Comment:
could you change the script to run here?

Reply:
Moved code to `real_convolve.py`. Added `signal` to the `defaults` config because `instantiate` required it. Therefore code runs both for the `defaults` and `exp1`.

Reply:
but the command for this config should `python examples/real_convolve.py -cn instantiate`?

Reply:
Oh, sorry, I forgot to delete this config, it is not useful anymore. The code works with both `defaults` and `exp1`. Signal class can be changed in command-line.